mlflow/pyfunc/model.py

```diff
@@ -825,198 +824,198 @@ def _maybe_decompress_cloudpickle_load(path, compression):
         return cloudpickle.load(f)
 
 
-if IS_PYDANTIC_V2_OR_NEWER:
-    from mlflow.types.responses import (
-        Message,
-        OutputItem,
-        ResponsesAgentRequest,
-        ResponsesAgentResponse,
-        ResponsesAgentStreamEvent,
-        create_annotation_added,
-        create_function_call_item,
-        create_function_call_output_item,
-        create_reasoning_item,
-        create_text_delta,
-        create_text_output_item,
-        output_to_responses_items_stream,
-        responses_agent_output_reducer,
-        responses_to_cc,
-        to_chat_completions_input,
-    )
+from mlflow.types.responses import (
```

Comment:
🔑 **Key change**: The `ResponsesAgent` class and related imports are no longer behind a Pydantic v2 guard. This entire block (200+ lines) was previously conditionally imported only when `IS_PYDANTIC_V2_OR_NEWER` was True.